package.json

```diff
@@ -62,7 +62,7 @@
     "@angular/platform-server": "17.0.4",
     "@angular/router": "17.0.4",
     "@stackblitz/sdk": "1.9.0",
-    "maplibre-gl": "3.6.2",
+    "maplibre-gl": "^3.6.2",
```

Comment:
As mentioned in my other comment, the dependencies in our package.json are pinned. This is intentional for transparency with renovate.
But you can use a range in the src/*/package.json files.

Reply:
> As mentioned in my other comment, the dependencies in our package.json are pinned. This is intentional for transparency with renovate. But you can use a range in the src/*/package.json files.

@kyubisation I tried fixing versions like you said. let me know if this is correct.